overlay.nix

```diff
@@ -49,7 +31,9 @@ let
 
     in
     {
-      "${subset}" = optUpstream // {
+      "${subset}" = optUpstream
+        // (qlib.pkgs-by-name callPackage ./pkgs/by-name)
```

Comment:
Should this be `lib.attrsets.recursiveUpdate` to avoid some surprises maybe?

Reply:
I was thinking about that. The attribute sets are relatively flat, with the exception of python that that gets handled separately. I would rather expect to get surprises with `recursiveUpdate`.

Reply:
Hu, could also be, yes :sweat_smile: The language subsets are strange, same with Haskell. Then let's see if this works as expected or someone complains later.